src/common/replication_common.cpp

```diff
@@ -591,7 +591,7 @@ replication_options::check_if_in_black_list(const std::vector<std::string> &blac
     return false;
 }
 
-const std::string replica_envs::DENY_CLIENT_WRITE("replica.deny_client_write");
+const std::string replica_envs::DENY_CLIENT("replica.deny_client");
```

Comment:
Keep in mind this will cause imcompatiable issue, there may be some community users use `replica.deny_client_write`.

Reply:
Yeah, I ever consider whether keep it. Maybe I need add some `hint_message` when user use `replica.deny_client_write`